File: file_compare/compare.py
with open('staging.env') as f:
    a = f.read()
    print(repr(a))


with open('staging_error.env') as f:
    b = f.read()
    print(repr(b))


print(a == b)


# 以 b 的形式读取，可以发现两者内容并不相同
with open('staging.env', 'rb') as f:
    a = f.read()
    print(repr(a))


with open('staging_error.env', 'rb') as f:
    b = f.read()
    print(repr(b))


print(a == b)

File: file_compare/compare.go
package main

import (
	"fmt"
	"io/ioutil"
)

func main() {
	a, _ := ioutil.ReadFile("staging.env")
	b, _ := ioutil.ReadFile("staging_error.env")
	fmt.Printf("%v\n", a)
	fmt.Printf("%v\n", b)
// 	for i, _ := range a {
// 		if a[i] != b[i] {
// 			fmt.Println(i, a[i], b[i])
// 		}
// 	}

	fmt.Println(string(a) == string(b))
}
